### PR TITLE
Delete sites

### LINKED ISSUE #68: Deleting a site
Routes:
- deleting a site deletes all students within that site (do not do this through the schema)

### PR DESCRIPTION
**Pull request by joshualchan**

### CODE REVIEW

File: routes/sites.js [hunk 1]
```diff
@@ -107,7 +114,7 @@ router.post('/', async (req, res) => {
     isNumeric(areaId, 'Area Id must be a Number');
     isPhoneNumber(primaryContactInfo.phone, 'Invalid Primary Phone Number');
     isBoolean(active, 'Active is not a boolean');
-    if (secondContactInfo.phone) {
+    if (secondContactInfo) {
```
Comment (anikait-rao) [inline routes/sites.js]: what's the purpose of this change

File: routes/sites.js [hunk 2]
```diff
@@ -280,24 +287,28 @@ router.put('/:siteId', async (req, res) => {
...
-        keysToCamel(
-          [site.rows[0]].map((s) =>
-            s.secondContactInfo && s.secondContactInfo.firstName
-              ? s
-              : { ...s, secondContactInfo: null },
-          ),
-        ),
-      );
+
+    res.status(200).send(keysToCamel(site.rows[0]));
```
Comment (anikait-rao) [inline routes/sites.js:317]: Also curious about this change

**Review by anikait-rao — CHANGES_REQUESTED** [review]
see comments

File: routes/sites.js (hunk 1 above)
Comment (joshualchan) [reply routes/sites.js]: There's an error if secondContactInfodoesn't exist (no attribute phone on undefined)

File: routes/sites.js (hunk 2 above)
Comment (joshualchan) [reply routes/sites.js:317]: think I was getting an error before (map not being a valid function bc the thing map was being called on wasn't a list)

**Review by anikait-rao — APPROVED** [review]
looks good
